### DISCUSSION TITLE
有关真实数据订单的问题

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/community-china#166] [存储方案]
你可以在这里发布有关真实数据的问题，主要是针对storage providers

**Comment by E0001** [comment]
![image](https://user-images.githubusercontent.com/57853534/137692638-713a8739-fb99-4986-a759-5752ecf0dd2f.png)

请问一下，miner按照文档配置之后，这里一直卡在查询价格这里，应该怎么解决这个问题呢

**Reply by de-authority** [reply]
一般情况下卡在这里有两个可能的原因
１：　网络问题，在客户端查询其它矿工的报价，看看是不是正常，如果别的矿工正常报价，则就是storage provider有问题
２：　storage provider 的公网ip没有暴露，可以修改config.toml中的libp2p区域，同时更新链上地址

**Reply by KaixuanZhao** [reply]
IPFS主网最大雅典娜矿池，RRmine global ，一站式采购托管运营，全部数据链上可查，全网算力领先1400PB，需要参与Datacap真实数据存储的朋友可以在小狐狸浏览器中复制打开https://www.rrmglobal.io/#/invite?invite_code=O3DKS2S0 欢迎加入社群 Wechet：kkmmjjnn00

**Reply by kimtony** [reply]
请问一下，这个是什么方式进行接单呢，是bidbot吗

**Reply by lee620422** [reply]
FILECOIN联合挖矿，算力销售，节点可查，香港IDC机房，安全有保障。微信滴滴：HBT_CNN

**Reply by lee620422** [reply]
再好的项目，布道最重要，所以……

**Reply by wengzhenqi** [reply]
ds数据能在新号上跑吗

**Comment by caijian76** [comment]
开启订单模式，对proivider的带宽有要求吗？

**Reply by de-authority** [reply]
没有明确要求，但是很明显，在线订单需要一定的带宽。建议根据实际需要扩充带宽。

**Comment by dcshy** [comment]
fil存量迁移出海 增量海外机房上架封装 独立节点集群解决方案 欢迎合作交流 v：：IPFS-Shy

**Reply by de-authority** [reply]
请不要发布虚假或者垃圾信息

**Comment by KaixuanZhao** [comment]
IPFS主网最大雅典娜矿池，提供一站式采购托管运营，fil存量数据迁移出海等服务，全网算力领先1400PB，需要B端1PB以上，C端50T，100T，200T（可扩容）和FIL存量数据迁移出海服务方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Comment by Chenhongwe** [comment]
Fil算力填充，240/T，包半年运维，送七牛商业存储软件一年，坐标杭州，欢迎骚扰 phone：15255579473（同V）

**Comment by LanmpPython** [comment]
请问怎么导入大文件，我按官方的文档操作不行。
![image](https://user-images.githubusercontent.com/45307601/149137268-64b5460a-0bd7-4a37-97af-c9c5db90f475.png)

**Comment by gdkiddd** [comment]
FILECOIN合法吗,有真实数据存储吗?https://zhuanlan.zhihu.com/p/482446614

**Reply by de-authority** [reply]
nice

**Comment by ChaiWenyu1983** [comment]
> 一般情况下卡在这里有两个可能的原因 １：　网络问题，在客户端查询其它矿工的报价，看看是不是正常，如果别的矿工正常报价，则就是storage provider有问题 ２：　storage provider 的公网ip没有暴露，可以修改config.toml中的libp2p区域，同时更新链上地址

我公网IP暴露了，也自己给自己发了个数据能存储，但是只能CC，接不到真实数据订单是什么情况？ 我是lotus daemon + lotus miner的形式，是因为市场没有分离吗，还是这个模式之能CC ，接订单的安装其他程序？

**Comment by K-Noir** [comment]
测试网谁有测试过吗，怎么申请测试网的十倍算力啊，求助！
